app/views/application/_head.haml

```diff
@@ -4,6 +4,8 @@
 %meta{charset: "utf-8"}/
 = content_for?(:meta_data) ? yield(:meta_data) : metas_tags
 
+%meta{:content => "yes", :name => "mobile-web-app-capable"}/
```

Comment:
Our style-check is currently down, but you can run it locally with `pronto run -c develop`:
```
app/views/application/_head.haml:7 W: Use the new Ruby 1.9 hash syntax.
```
So the line should look like this:
```
%meta{content: "yes", name: "mobile-web-app-capable"}/
```